virttest/test_setup.py

```diff
@@ -2225,9 +2225,9 @@ def remote_session(params):
     :param params: Test params dict for remote machine login details
     :return: remote session object
     """
-    server_ip = params["server_ip"]
-    server_user = params.get("server_user", "root")
-    server_pwd = params["server_pwd"]
+    server_ip = params["remote_ip"]
+    server_user = params.get("remote_user", "root")
```

Comment:
@dzhengfy, can we default to param `remote_*` something like,
```python
server_ip = params.get(params.get("server_ip"), params['remote_ip'])
server_user = params.get(params.get("server_user"), params['remote_user'])
server_pwd = params.get(params.get("server_pwd"), params['remote_pwd'])
```

Reply:
@balamuruhans , I like this idea, which can benefit both of scenarios. I will do the update.